interp/notation.ml

```diff
@@ -348,6 +353,15 @@ let notation_constr_key = function (* Rem: NApp(NRef ref,[]) stands for @ref *)
 type level = notation_entry * entry_level * entry_relative_level list
   (* first argument is InCustomEntry s for custom entries *)
 
+let entry_relative_level_le child parent =
```

Comment:
Can you add a short explanation of what the function does? It is not immediately obvious to me e.g. why `child` is an option.

Reply:
I'll do. Maybe will I be able to find slightly more explicit invariants actually.